brainscore_vision/model_helpers/activations/core.py

```diff
@@ -209,17 +340,34 @@ def _package_layer(self, layer_activations, layer, stimuli_paths):
             self._logger.debug(f"Unknown layer activations shape {layer_activations.shape}, not inferring channels")
 
         # build assembly
-        coords = {'stimulus_path': stimuli_paths,
+        coords = {'stimulus_path': ('presentation', stimuli_paths),
+                  'microsaccade_shift_x_pixels': ('presentation', self.unpack_microsaccade_coords(
+                      np.unique(stimuli_paths),
+                      pixels_or_degrees='pixels',
+                      dim=0)),
+                  'microsaccade_shift_y_pixels': ('presentation', self.unpack_microsaccade_coords(
+                      np.unique(stimuli_paths),
+                      pixels_or_degrees='pixels',
+                      dim=1)),
+                  'microsaccade_shift_x_degrees': ('presentation', self.unpack_microsaccade_coords(
+                      np.unique(stimuli_paths),
+                      pixels_or_degrees='degrees',
+                      dim=0)),
+                  'microsaccade_shift_y_degrees': ('presentation', self.unpack_microsaccade_coords(
+                      np.unique(stimuli_paths),
+                      pixels_or_degrees='degrees',
+                      dim=1)),
```

Comment:
please add a comment explaining what is going on here. This should probably be extracted into a method `_build_microsaccade_coords`, i.e.
```
coords = {**{'stimulus_path': ('presentation', stimuli_paths),
                 **self._build_microsaccade_coords(...),
                'neuroid_num': ('neuroid', ...
                ...
                } 
```